frontend/src/__tests__/cypress/cypress/e2e/pipelines/Pipelines.cy.ts

```diff
@@ -116,9 +116,172 @@ describe('Pipelines', () => {
     cy.wait('@refreshVersions');
 
     // Verify the uploaded pipeline version is in the table
-    pipelinesTable.toggleExpandRowByIndex(0);
+    pipelinesTable.toggleExpandRowByName(initialMockPipeline.name);
     pipelinesTable.findRowByName('New pipeline version');
   });
+
+  it('delete single pipeline', () => {
+    let deleted = false;
+
+    initIntercepts();
+    initDeletePipelineIntercept(initialMockPipeline.id);
+    cy.intercept(
+      {
+        pathname: '/api/proxy/apis/v1beta1/pipelines',
+      },
+      (req) => {
+        req.reply(buildMockPipelines(deleted ? [] : [initialMockPipeline]));
+      },
+    );
+    cy.intercept(
+      {
+        method: 'POST',
+        pathname: '/api/proxy/apis/v1beta1/pipeline',
```

Comment:
Should this pathname be returning `buildMockPipelineVersions`? I would expect this to be `api/proxy/apis/v1beta1/pipeline_versions`, no?